src/content/docs/fr/reference/api-reference.mdx

```diff
@@ -0,0 +1,1948 @@
+
+##### `headers`
+
+<p>
+**Type :** `() => Iterator<string>`
+</p>
+
+Obtient les valeurs de l'en-tête `Set-Cookie` qui seront envoyées avec la réponse.
+
+#### `AstroCookie`
+
+Obtenir un cookie via `Astro.cookies.get()` renvoie un type `AstroCookie`. Il a la structure suivante.
+
+##### `value`
+
+<p>
+**Type :** `string | undefined`
+</p>
+
+La valeur brute de la chaîne du cookie.
+
+##### `json`
+
+<p>
+**Type :** `() => Record<string, any>`
+</p>
+
+Analyse la valeur du cookie via `JSON.parse()`, retournant un objet. Lance si la valeur du cookie n'est pas un JSON valide.
+
+##### `number`
+
+<p>
+**Type :** `() => number`
+</p>
+
+Analyse la valeur du cookie en tant que nombre. Renvoie NaN s'il ne s'agit pas d'un nombre valide.
+
+##### `boolean`
+
+<p>
+**Type :** `() => boolean`
+</p>
+
+Convertit la valeur du cookie en un booléen.
+
+#### `CookieGetOptions`
+
+<p><Since v="4.1.0"/></p>
+
+L'obtention d'un cookie permet également de spécifier des options via l'interface `CookieGetOptions` :
+
+##### `decode`
+
+<p>
+**Type :** `(value: string) => string`
+</p>
+
+Permet de personnaliser la manière dont un cookie est désérialisé en une valeur.
+
+#### `CookieSetOptions`
+
+<p><Since v="4.1.0"/></p>
+
+La définition d'un cookie via `Astro.cookies.set()` permet de passer un `CookieSetOptions` pour personnaliser la façon dont le cookie est sérialisé.
+
+##### `domain`
+
+<p>
+**Type :** `string`
+</p>
+
+Spécifie le domaine. Si aucun domaine n'est défini, la plupart des clients interpréteront l'application au domaine actuel.
+
+##### `expires`
+
+<p>
+**Type :** `Date`
+</p>
+
+Spécifie la date d'expiration du cookie.
+
+##### `httpOnly`
+
+<p>
+**Type :** `boolean`
+</p>
+
+Si c'est le cas, le cookie ne sera pas accessible côté client.
+
+##### `maxAge`
+
+<p>
+**Type :** `number`
+</p>
+
+Spécifie un nombre, en secondes, pour lequel le cookie est valide.
+
+##### `path`
+
+<p>
+**Type :** `string`
+</p>
+
+Spécifie un sous-chemin du domaine dans lequel le cookie est appliqué.
+
+##### `sameSite`
+
+<p>
+**Type :** `boolean | 'lax' | 'none' | 'strict'`
+</p>
+
+Spécifie la valeur de l'en-tête du cookie [SameSite](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-09#section-5.4.7).
+
+##### `secure`
+
+<p>
+**Type :** `boolean`
+</p>
+
+If true, the cookie is only set on https sites.
+
+##### `encode`
+
+<p>
+**Type :** `(value: string) => string`
+</p>
+
+Permet de personnaliser la façon dont le cookie est sérialisé.
+
+### `Astro.redirect()`
+
+Permet de rediriger vers une autre page, et optionnellement de fournir un [code de réponse avec un statut HTTP](https://developer.mozilla.org/fr/docs/Web/HTTP/Status) comme second paramètre.
+
+Une page (et non un composant enfant) doit `return` le résultat de `Astro.redirect()` pour que la redirection ait lieu.
```

Comment:
In this case, I think we can replace the inline code with a French word to be more readable:
```suggestion
Une page (et non un composant enfant) doit retourner le résultat de `Astro.redirect()` pour que la redirection ait lieu.
```